prow/cmd/exporter/collector_test.go

```diff
@@ -19,8 +19,17 @@ package main
 import (
 	"reflect"
 	"testing"
+	"time"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/util/diff"
+
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
```

Comment:
Why is this dto?

Reply:
blindly copied from upstream :)

https://github.com/prometheus/client_golang/blob/9a37535b1ba53f6216e9c8557559a714e0dbb84a/prometheus/value.go#L22